forgerock-auth/src/main/java/org/forgerock/android/auth/Config.java

```diff
@@ -204,8 +213,9 @@ TokenManager getTokenManager() {
 
     SingleSignOnManager getSingleSignOnManager() {
         return DefaultSingleSignOnManager.builder()
-                .sharedPreferences(ssoSharedPreferences)
```

Comment:
i am trying to understand this. we had one storage before for both SSO and cookies ?